src/components/header/index.tsx

```diff
@@ -5,12 +5,12 @@ import { Logo } from './logo'
 
 export function Header() {
   return (
-    <header className='bg-ic-white shadow-ic-black/15 sticky top-0 z-50 flex justify-between px-9 py-4 opacity-[.96] shadow-md backdrop-blur'>
+    <header className='bg-ic-white dark:bg-ic-blue-950 shadow-ic-black/15 sticky top-0 z-50 flex justify-between px-9 py-4 opacity-[.96] shadow-md backdrop-blur'>
       <div className='flex flex-row items-center space-x-6'>
         <Logo />
         <HeaderLink href={Path.TRADE} label='Trade' />
         <HeaderLink href={Path.PRODUCTS} label='Products' />
-        {/* <HeaderLink href={Path.LEVERAGE} label='Leverage' /> */}
+        <HeaderLink href={Path.LEVERAGE} label='Leverage' />
```

Comment:
Hide again, before merging ⚠️

Reply:
@janndriessen what about adding a check against the vercel environment to determine if we should show it?

Reply:
https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNGMwMDA0aXI2dHM0cnZmd2R3d2s3aHgzbWNtZGZ1NHZtMnhxaW5rcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/R6gvnAxj2ISzJdbA63/giphy-downsized-large.gif